src/fortran_routing/mc_pylink_v00/Reservoir_singleTS/reservoirs_nwm.py

```diff
@@ -0,0 +1,110 @@
+import traceback
+
+debuglevel = 0
+COMPILE = True
+if COMPILE:
+    try:
+        import subprocess
+
+        fortran_compile_call = []
+        fortran_compile_call.append(r"f2py3")
+        fortran_compile_call.append(r"-c")
+        fortran_compile_call.append(r"varPrecision.f90")
+        fortran_compile_call.append(r"module_levelpool.f90")
+        fortran_compile_call.append(r"-m")
+        fortran_compile_call.append(r"levelpools")
+        # fortran_compile_call.append(r"--opt='-fdefault-real-8'")
+
+        if debuglevel <= -1:
+            print(fortran_compile_call)
+        if debuglevel <= -2:
+            subprocess.run(fortran_compile_call)
+        else:
+            subprocess.run(
+                fortran_compile_call,
+                stdout=subprocess.DEVNULL,
+                stderr=subprocess.DEVNULL,
+            )
+        from levelpools import *
+
+        # import mc_sseg_stime as muskingcunge_module
+    except Exception as e:
+        print(e)
+        if debuglevel <= -1:
+            traceback.print_exc()
+else:
+    from levelpools import *
+
+    # import mc_sseg_stime as muskingcunge_module
+
+
+def reservoirs_calc(
+    dt=None,
+    qi0=None,
+    qi1=None,
+    ql=None,
+    ar=None,
+    we=None,
+    maxh=None,
+    wc=None,
+    wl=None,
+    dl=None,
+    oe=None,
+    oc=None,
+    oa=None,
+    h0=None,
+):
+    """This test function demonstrates a simple connection between
```

Comment:
I updated this python script to be a very brief demonstration of how the module is used with f2py.